src/main/java/uk/gov/ons/ctp/response/collection/exercise/distribution/SampleUnitDistributor.java

```diff
@@ -289,11 +293,40 @@ private CollectionExercise collectionExerciseTransitionState(CollectionExercise
 
     try {
       if (published == exercise.getSampleSize().longValue()) {
-        // All sample units published, set exercise state to PUBLISHED
-        exercise.setState(
-            collectionExerciseTransitionState.transition(
-                exercise.getState(), CollectionExerciseDTO.CollectionExerciseEvent.PUBLISH));
-        exercise.setActualPublishDateTime(new Timestamp(new Date().getTime()));
+
+        Boolean isGoLiveInPast = Boolean.FALSE;
```

Comment:
generally it's preferred to use boolean primitive instead of the Boolean class.

Also, you could just put the if first if condition as the clause for the second and not have the local variable at all